docs/source/stubtest.rst

```diff
@@ -0,0 +1,136 @@
+.. _stubtest:
+
+.. program:: stubtest
+
+Automatic stub testing (stubtest)
+=================================
+
+Stub files are files containing type annotations. See
+`PEP 484 <https://www.python.org/dev/peps/pep-0484/#stub-files>`_
+for more motivation and details.
+
+A common problem with stub files is that they tend to diverge from the
+actual implementation. Mypy includes the ``stubtest`` tool that can
+automatically check for discrepancies between the stubs and the
+implementation at runtime.
+
+What stubtest does and does not do
+**********************************
+
+stubtest will import your code and introspect your code objects at runtime, for
```

Comment:
Feels a bit weird to start a sentence with lowercase "stubtest". Maybe recast the sentence to not put it at the beginning, or consistently write ``stubtest``.